.github/workflows/ci.yml

```diff
@@ -642,6 +642,11 @@ jobs:
       env:
         DD_API_KEY: ${{ secrets.DD_API_KEY }}
         DD_APPLICATION_KEY: ${{ secrets.DD_APPLICATION_KEY }}
+        DD_APP_KEY: ${{ secrets.DD_APPLICATION_KEY }}
+        DD_API_KEY_2: ${{ secrets.DD_API_KEY_2 }}
+        DD_APP_KEY_2: ${{ secrets.DD_APP_KEY_2 }}
```

Comment:
Looks like `secrets.DD_APP_KEY_2` is not the correct one: https://github.com/DataDog/system-tests/actions/runs/4821436378/jobs/8587339864?pr=1016

@cbeauchesne should I use `secrets.DD_APPLICATION_KEY_2` instead?

Reply:
Ci relaucnhed